src/test/java/org/openrewrite/java/logging/slf4j/Slf4jLogShouldBeConstantTest.java

```diff
@@ -199,4 +206,63 @@ void method() {
           )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/95")
```

Comment:
Again; thanks for adding these in immediately. :)

Reply:
you are welcome!